readthedocs_ext/_static/readthedocs-data.js_t

```diff
@@ -11,6 +11,7 @@ var READTHEDOCS_DATA = {
     source_suffix: "{{ source_suffix }}",
     api_host: "{{ api_host }}",
     commit: "{{ commit }}",
+    ad_free: {% if ad_free %}true{% else %}false{% endif %},
```

Comment:
Should this take into account gold status as well, since that has the same effect?

Reply:
Ah, just saw #4329 👍